### ISSUE TITLE
Why is there no biological examples of plots in R?

### ISSUE THREAD

**Issue by CraigSteinberg** [aodn/imos-user-code-library#18]

**Comment by ggalibert** [comment]
@xhoenner would you have some ready made code that you could contribute to this repo? I'm thinking about the seal data kin of plots/animation that you did in the past.

**Comment by ggalibert** [comment]
And maybe also AusCPR, BGC analysis on water samples, etc... just in case you had something for this.

**Comment by ggalibert** [comment]
Also:
-NRS zooplankton and phytoplankton abundance / biomass
-Australian zooplankton / phytoplankton database

**Comment by xhoenner** [comment]
Agreed that some examples for that community are needed, the results of Jason Everett's survey may shed some light on what biologists are after.

A few ideas that came to mind, which might be worth considering, and for which I already have functional prototypes in R:
 * Example of WFS requests to retrieve IMOS data through Geoserver
 * Extracting physical observations (SST, sea surface current, chlorophyll a) at points along a track
